java/group-layers/README.md

```diff
@@ -2,19 +2,25 @@
 
 Group a collection of layers together and toggle their visibility as a group.
 
-![Group layers app](group-layers.png)
+![Image of group layers](group-layers.png)
 
-## Use case
+## Use case  
 
 Group layers communicate to the user that layers are related and can be managed together.
 
 In a land development project, you might group layers according to the phase of development.
 
+## How to use the sample
+
+The layers in the map will be displayed in a table of contents. Toggle the checkbox next to a layer's name to change its visibility. Turning a group layer's visibility off will override the visibility of its child layers.
+
 ## How it works
 
-1. Create an empty `GroupLayer`
-1. Add a child layer to the group layer's layers collection.
-1. To toggle the visibility of the group, simply change the group layer's visibility property.
+1. Add the `GroupLayer` and additional `Layers` as operational layers in the `SceneView`.
```

Comment:
We should use our API class names as they are within backticks, suggestion below
```suggestion
1. Add the `GroupLayer` and additional `Layer`s as operational layers in the `SceneView`.
```